### ISSUE TITLE
Run python tests on plasma framework

### ISSUE THREAD

**Issue by pik694** [omisego/plasma-contracts#150]
In #116, we decided to  rewrite all current tests into `js`, but in order to ensure that the rewrite is secure and works as intended we want the old python tests to run on the new contracts. 

Once we have rewritten all the tests we will eventually delete python tests, but for now we should keep them, but the tests should now test `plasma framework` instead of `RootChain`.

**Comment by bg1156** [comment]
*but the tests should now test plasma framework instead of RootChain.*

*just curious*, so is the root chain no longer needed in the framework?
tldr would be nice, note i am not a dev~

**Comment by bg1156** [comment]
*just curious*, so is the root chain no longer needed in the framework?
tldr would be nice, note i am not a dev~

**Comment by pik694** [comment]
We are in progress of refactoring the old contract which was called `RootChain`. 
Instead of having one enormous contract (and file),  we divide it into maintainable, readable pieces, which altogether are called `plasma framework`. 

When I wrote :
>  instead of RootChain
I made a mental shortcut and also it is how we internally differentiate those two versions. 

What I should have written was that we will switch python tests from testing the old version of the contract into the new one.

**Comment by bg1156** [comment]
thank you, just trying to keep up

**Comment by pik694** [comment]
🎉

**Comment by boolafish** [comment]
🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳
